src/core/api/quota/migrator.go

```diff
@@ -111,6 +123,10 @@ func Sync(pm promgr.ProjectManager, populate bool) error {
 	return nil
 }
 
+func Check(f func() error) error {
+	return retry(10, 100*time.Millisecond, f)
```

Comment:
first retry will happen in 0.1 second?

Reply:
no, please just see the retry func details.

Reply:
the data of about ten times,

123.889705ms
341.617085ms
782.73897ms
2.152148225s
4.482828918s
12.274047382s
32.847927757s
1m21.819425406s
3m52.175972572s